### DISCUSSION TITLE
Image preloading - experiment failed to load

### DISCUSSION THREAD

**Discussion by synapticlee** [jspsych/jsPsych#2948] [Q&A] [answered]
jspsych 6.1.0

Hi, I'm trying to preload 30 stimuli, and I'm doing this by calling a .js script called stimuli.js at the top of my HTML, which loads a var called images. 

I'm then trying to preload these with, 
    jsPsych.init({
      timeline: timeline,
      preload_images: images,
      on_finish: function() {
        jsPsych.data.displayData(); // comment out when live
      }
    });

And nothing else. However when I run the experiment it says "the experiment failed to load". I only have 30 stimuli, so I'm not sure why this is a problem. Can anyone help?

I decided to preload as some of my stimuli were showing up with a broken link image when testing my experiment.

**Comment by jodeleeuw** [comment]
Hi @synapticlee,

If you are getting a broken link image running the experiment this indicates that the path to the image file is incorrect. This would also explain why the experiment is failing to load: if there are files that cannot be found then preloading will fail. If you open the developer tools and/or console in your browser, you should see error messages for each failed image load and that will help you track down which paths are incorrect.

**Comment by synapticlee** [comment, marked as answer]
Hi, that's why I'm confused as the stimulus path that shows up in the JSON
at the end of the experiment is correct. Is there any other reason this
could be happening?

**Reply by jodeleeuw** [reply]
Very unlikely that it is another reason. Are there errors in the developer console? There is probably a typo or small difference in one or more of the filenames that is causing the issue.

**Reply by synapticlee** [reply]
There are no errors in the developer console and I've triple checked that the stimulus is there, even deleting it and putting it back in the same folder

**Reply by synapticlee** [reply]
I remade the stimulus in photoshop and somehow it works now..
